### PR TITLE
update multi-lang doc

### PR DESCRIPTION
**Pull request by tink2123**

### CODE REVIEW

File: doc/doc_ch/recognition.md [hunk 1]
```diff
@@ -272,16 +270,109 @@ Eval:
...
+    cd PaddleOCR/configs/rec/multi_language/
+    # 通过-l或者--language参数设置需要生成的语种的配置文件，该命令会将默认参数写入配置文件
+    python3 generate_multi_language_configs.py -l it
+    ```
+
+- 如果您的数据放置在其他位置，或希望使用自己的字典，可以通过指定相关参数来生成配置文件:
+
+    ```bash
+    # -l或者--language字段是必须的
+    # --train修改训练集，--val修改验证集，--data_dir修改数据集目录，-o修改对应默认参数
```
Comment (xmy0916) [inline doc/doc_ch/recognition.md:306]: 这里少了--dict修改字典路径的注释

File: doc/doc_ch/recognition.md [hunk 2]
```diff
@@ -272,16 +270,109 @@ Eval:
...
+      ...
+
+   ```
+
+目前PaddleOCR支持的多语言算法有：
+
+| 配置文件 |  算法名称 |   backbone |   trans   |   seq      |     pred     |  language | character_type |
+| :--------: |  :-------:   | :-------:  |   :-------:   |   :-----:   |  :-----:   | :-----:  | :-----:  |
+| rec_chinese_cht_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 中文繁体  | chinese_cht|
+| rec_en_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 英语   | ch |
```
Comment (xmy0916) [inline doc/doc_ch/recognition.md:351]: 这里英语的type为ch是不是写错了~

File: doc/doc_en/recognition_en.md [hunk 3]
```diff
@@ -266,15 +266,115 @@ Eval:
...
+    ```
+
+- If your data is placed in another location, or you want to use your own dictionary, you can generate the configuration file by specifying the relevant parameters:
+
+    ```bash
+    # -l or --language field is required
+    # --train to modify the training set
+    # --val to modify the validation set
+    # --data_dir to modify the data set directory
+    # -o to modify the corresponding default parameters
```
Comment (xmy0916) [inline doc/doc_en/recognition_en.md:307]: 少了--dict的注释

File: doc/doc_en/recognition_en.md [hunk 4]
```diff
@@ -266,15 +266,115 @@ Eval:
...
+        name: SimpleDataSet
+        data_dir: train_data/ # root directory of val data
+        label_file_list: ["./train_data/val_list.txt"] # val label path
+      ...
+
+   ```
+
+Currently, the multi-language algorithms supported by PaddleOCR are:
+
+| 配置文件 |  算法名称 |   backbone |   trans   |   seq      |     pred     |  language | character_type |
```
Comment (xmy0916) [inline doc/doc_en/recognition_en.md:349]: 这里应该是英文

File: doc/doc_en/recognition_en.md [hunk 5]
```diff
@@ -266,15 +266,115 @@ Eval:
...
+| :--------: |  :-------:   | :-------:  |   :-------:   |   :-----:   |  :-----:   | :-----:  | :-----:  |
+| rec_chinese_cht_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | chinese traditional  | chinese_cht|
+| rec_en_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | English   | ch |
```
Comment (xmy0916) [inline doc/doc_en/recognition_en.md:352]: type是en

File: doc/doc_en/inference_en.md [hunk 6]
```diff
@@ -318,7 +318,7 @@ If you need to predict other language models, when using inference model predict
 You need to specify the visual font path through `--vis_font_path`. There are small language fonts provided by default under the `doc/` path, such as Korean recognition:
```
Comment (xmy0916) [inline doc/doc_en/inference_en.md:318]: under the `doc/fonts` path

File: doc/doc_ch/recognition.md (hunk 1 above)
Comment (tink2123) [reply doc/doc_ch/recognition.md]: done, thx

File: doc/doc_ch/recognition.md (hunk 2 above)
Comment (tink2123) [reply doc/doc_ch/recognition.md]: fixed, thx

File: doc/doc_en/recognition_en.md (hunk 3 above)
Comment (tink2123) [reply doc/doc_en/recognition_en.md]: done, thx

File: doc/doc_en/recognition_en.md (hunk 4 above)
Comment (tink2123) [reply doc/doc_en/recognition_en.md]: done, thx

File: doc/doc_en/recognition_en.md (hunk 5 above)
Comment (tink2123) [reply doc/doc_en/recognition_en.md]: done, thx

File: doc/doc_en/inference_en.md (hunk 6 above)
Comment (tink2123) [reply doc/doc_en/inference_en.md]: done,thx

File: doc/doc_ch/recognition.md [hunk 7]
```diff
@@ -272,16 +270,109 @@ Eval:
...
+      ...
+
+   ```
+
+目前PaddleOCR支持的多语言算法有：
+
+| 配置文件 |  算法名称 |   backbone |   trans   |   seq      |     pred     |  language | character_type |
+| :--------: |  :-------:   | :-------:  |   :-------:   |   :-----:   |  :-----:   | :-----:  | :-----:  |
+| rec_chinese_cht_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 中文繁体  | chinese_cht|
+| rec_en_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 英语   | En |
```
Comment (LDOUBLEV) [inline doc/doc_ch/recognition.md:351]: 这个表格里没有en，只有En，二者的区别也要说清楚

File: ppocr/postprocess/rec_postprocess.py [hunk 8]
```diff
@@ -24,19 +24,26 @@ def __init__(self,
...
         assert character_type in support_character_type, "Only {} are supported now but get {}".format(
             support_character_type, character_type)
 
         if character_type == "en":
             self.character_str = "0123456789abcdefghijklmnopqrstuvwxyz"
             dict_character = list(self.character_str)
-        elif character_type in ["ch", "french", "german", "japan", "korean"]:
+        elif character_type == "en_sensitive":
+            # same with ASTER setting (use 94 char).
+            import string
```
Comment (LDOUBLEV) [inline ppocr/postprocess/rec_postprocess.py:40]: import 建议放文件开头

File: ppocr/data/imaug/label_ops.py [hunk 9]
```diff
@@ -101,9 +104,15 @@ def __init__(self,
         if character_type == "en":
             self.character_str = "0123456789abcdefghijklmnopqrstuvwxyz"
             dict_character = list(self.character_str)
-        elif character_type in ["ch", "french", "german", "japan", "korean"]:
+        elif character_type == "en_sensitive":
+            # same with ASTER setting (use 94 char).
+            import string
```
Comment (LDOUBLEV) [inline ppocr/data/imaug/label_ops.py:109]: import 放文件开头

File: doc/doc_en/recognition_en.md [hunk 10]
```diff
@@ -266,15 +266,116 @@ Eval:
...
+      ...
+
+   ```
+
+Currently, the multi-language algorithms supported by PaddleOCR are:
+
+| Configuration file |  Algorithm name |   backbone |   trans   |   seq      |     pred     |  language | character_type |
+| :--------: |  :-------:   | :-------:  |   :-------:   |   :-----:   |  :-----:   | :-----:  | :-----:  |
+| rec_chinese_cht_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | chinese traditional  | chinese_cht|
+| rec_en_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | English   | En |
```
Comment (LDOUBLEV) [inline doc/doc_en/recognition_en.md:353]: 同上

File: doc/doc_ch/recognition.md [hunk 11]
```diff
@@ -272,16 +270,109 @@ Eval:
...
+| rec_ne_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 尼泊尔语  | ne |
+| rec_rsc_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 塞尔维亚(cyrillic)语 |  rsc |
+| rec_bg_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 保加利亚语  | bg |
+| rec_uk_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 乌克兰语  | uk |
+| rec_be_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 白俄罗斯语   | be |
+| rec_te_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 泰卢固语  | te |
+| rec_ka_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 卡纳达语  | ka |
+| rec_ta_lite_train.yml |  CRNN |   Mobilenet_v3 small 0.5 |  None   |  BiLSTM |  ctc  | 泰米尔语 |  ta |
 
 多语言模型训练方式与中文模型一致，训练数据集均为100w的合成数据，少量的字体可以在 [百度网盘](https://pan.baidu.com/s/1bS_u207Rm7YbY33wOECKDA) 上下载，提取码：frgi。
```
Comment (xmy0916) [inline doc/doc_ch/recognition.md:377]: 这个字体的链接要不要更新下加上其他小语种的~

File: ppocr/postprocess/rec_postprocess.py [hunk 12]
```diff
@@ -24,19 +24,26 @@ def __init__(self,
                  character_type='ch',
                  use_space_char=False):
         support_character_type = [
-            'ch', 'en', 'en_sensitive', 'french', 'german', 'japan', 'korean', 'it',
-            'xi', 'pu', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur', 'rs', 'oc', 'rsc', 'bg',
-            'uk', 'be', 'te', 'ka', 'chinese_cht', 'hi', 'mr', 'ne'
+            'ch', 'en', 'en_sensitive', 'french', 'german', 'japan', 'korean',
```
Comment (WenmuZhou) [inline ppocr/postprocess/rec_postprocess.py:27]: 建议删掉en_sensitive，不然会和En有歧义

File: doc/doc_ch/recognition.md (hunk 7 above)
Comment (tink2123) [reply doc/doc_ch/recognition.md]: Done, thx

File: ppocr/postprocess/rec_postprocess.py (hunk 8 above)
Comment (tink2123) [reply ppocr/postprocess/rec_postprocess.py]: done, thx

File: ppocr/data/imaug/label_ops.py (hunk 9 above)
Comment (tink2123) [reply ppocr/data/imaug/label_ops.py]: done, thx

File: doc/doc_en/recognition_en.md (hunk 10 above)
Comment (tink2123) [reply doc/doc_en/recognition_en.md]: done, thx

File: doc/doc_ch/recognition.md (hunk 11 above)
Comment (tink2123) [reply doc/doc_ch/recognition.md:377]: 暂时不加了吧～

File: ppocr/postprocess/rec_postprocess.py (hunk 12 above)
Comment (tink2123) [reply ppocr/postprocess/rec_postprocess.py]: done, thx

**Review by WenmuZhou — APPROVED** [review]
